README.md

```diff
@@ -6,7 +6,7 @@ Pinp is not PNAS
 
 The template provided by the PNAS Article template in the
 [rticles](https://cran.r-project.org/package=rticles) package makes it very easy and convenient to
-create attractive looking two-column papers.
+create attractive looking one or two-column papers.
```

Comment:
The one thing that rubs me the wrong way here is that it lists one-columns as if we really cared about one column.

I don't think we do, really.  Pinp is about two-col defaults, with one-col thrown in _because why not_.  

That doesn't come out here.